features/serviceworker-blocking.json

```diff
@@ -0,0 +1,11 @@
+{
+    "_meta": {
+        "description": "Service worker blocking. Blocking of requests initiated by service workers",
+        "sampleExcludeRecords": {
+            "domain": "example.com",
+            "reason": "site breakage"
+        }
+    },
+    "state": "enabled",
```

Comment:
I don't think the state belongs in this file, but in the platform overrides (e.g. [extension-override.json](https://github.com/duckduckgo/privacy-configuration/blob/main/overrides/extension-override.json)). But thinking about it, perhaps we should only enable the feature for a more specific group first? I think we could enable it for only Firefox extension users for example.